### DISCUSSION TITLE
[Checkbox] Reset checkbox 'checked' state when form is reset using `form.reset()`

### DISCUSSION THREAD

**Discussion by rkumorek** [radix-ui/primitives#982] [Help]
Hey,

I'm wondering if it would be possible to reset Radix Checkbox checked state on `form.reset()` action, exactly as it works with plain <input type="checkbox"/> elements?

I am not sure if it's a limitation due to how React works or it's a bug or a feature request issue so apologies again if it's in wrong place.

You can see current behaviour in below code sandbox:
https://codesandbox.io/s/heuristic-orla-cnd7c

Thanks,
Rafal

**Comment by jjenzz** [comment]
Seems like a valid feature request to me. I'll convert this to an issue for us to investigate 👍

EDIT: Issue created here https://github.com/radix-ui/primitives/issues/983
